openmc/settings.py

```diff
@@ -1087,3 +1136,88 @@ def export_to_xml(self):
         tree = ET.ElementTree(self._settings_file)
         tree.write("settings.xml", xml_declaration=True,
                              encoding='utf-8', method="xml")
+
+
+class ResonanceScattering(object):
+    """Specification of the elastic scattering model for resonant isotopes.
+
+    Attributes
+    ----------
+    nuclide : openmc.nuclide.Nuclide
+        The nuclide affected by this resonance scattering treatment.
+    nuclide_0K : openmc.nuclide.Nuclide
+        This should be the same isotope as `nuclide`, but it should have an
```

Comment:
I thought it added a little extra clarity, but in retrospect it's a little weird to have this be the only place where that convention is used.  I've removed them.